modules/dbnd-airflow/src/dbnd_airflow/airflow_extensions/request_factory.py

```diff
@@ -1,38 +1,67 @@
-from airflow.contrib.kubernetes.kubernetes_request_factory.pod_request_factory import (
-    SimplePodRequestFactory as AirflowSimplePodRequestFactory,
-)
+import logging
+
+from distutils.version import LooseVersion
+
+import airflow
+import yaml
+
 from airflow.contrib.kubernetes.secret import Secret
 
+from dbnd_airflow.compat.request_factory import serialize_pod
+
+
+logger = logging.getLogger(__name__)
+
+
+class DbndPodRequestFactory(object):
```

Comment:
if this class is not in use anymore by airflow directly -> we should start to move it logic into KubernetesEngineConfig